components/input/input.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sinister-UI | Input</title>
+    <link rel="icon" type="image/x-icon" href="../../assets/Sinister-UI.svg">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="../../components/main.css">
+    <!-- <link rel="stylesheet" href="../../components/typography/typography.css"> -->
+    <link rel="stylesheet" href="../../docs.css">
+    <link rel="stylesheet" href="../../landing.css">
+</head>
+<body>
+    <header class="navbar" >
+        <p class="navbar-brand">
+            <a href="../../index.html"  >Sinister UI</a>
+            <small class="subtitle2 small-text" >V1.0</small>
+        </p>        
+        <nav>
+            <a href="../../docs.html" class="btn btn-text teal" >Docs</a>        
+        </nav> 
+
+    </header>
+    <main class="docs-container">
+        <aside class="sidebar">
+            <ul>
+                <span class="headline4">Get Started</span>
+                <a href="#" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Installation
+                    </li>
+                </a>
+                <a href="#" class="btn btn-text btn-link teal button-txt" >
+                    <li>
+                        Color
+                    </li>
+                </a>
+                <a href="#" class="btn btn-text btn-link teal button-txt" >
```

Comment:
You have used both `btn-text` and  `button-text`.
It seems similar.
Use proper naming conventions.